index.css

```diff
@@ -0,0 +1,64 @@
+
+
+*{
+    margin: 0%;
+    padding: 0%;
+    box-sizing: border-box;
+}
+h1{
+    font-family: "Inter";
+    font-weight: 400;
+    margin: 1rem 0px 0px 1rem;
+}
+.todos {
+    margin: 1rem 0px 0px 1rem;
+}
+
+.todoList {
+    display: flex;
+    gap: 15px;
+    background-color: rgb(244, 244, 134);
+    padding: 10px;
+    min-width: 250px;
+    max-width: 400px;
+}
+.todoList:nth-child(odd){
```

Comment:
Not sure if this was confusing in the Figma, but the idea was that only the first one should be red and the second should be yellow, rest should have no background. -1